push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/NotificationDispatcher.java

```diff
@@ -50,27 +49,41 @@
     @Inject
     private PushMessageMetricsService pushMessageMetricsService;
 
+    private final String ADM_TOPIC = "AdmToken_Topic";
```

Comment:
Use "agpush" prefix. Can u take a look how other kafka topics are named so we can keep them consistent?